GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs

```diff
@@ -12,24 +12,12 @@ namespace GoogleMapsApi.Test.IntegrationTests
     public class BaseTestIntegration
     {
         const string ApiKeyEnvironmentVariable = "GOOGLE_API_KEY";
-        private readonly IConfigurationRoot Configuration;
 
         public BaseTestIntegration()
         {
-            var builder = new ConfigurationBuilder()
-                .SetBasePath(Directory.GetCurrentDirectory())
-                .AddEnvironmentVariables();
```

Comment:
@energywave I think that you're missing this line. 
What was the purpose of this change?